examples/facebook/.eslintrc.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   parserOptions: {
     allowImportExportEverywhere: true,
   },
-  extends: ['airbnb', 'prettier', 'prettier/react'],
```

Comment:
@Josh-Cena I missed this but apparently the Facebook template was modified in https://github.com/facebook/docusaurus/pull/5963

Do you remember why? I don't think it's a big deal in this case, but I'm not sure how we should make this template evolve over time so I'd rather keep it as is unless someone ask for a modification

Reply:
Yes, in `eslint-config-prettier@8`, the `prettier` config also includes `prettier/react`. https://github.com/prettier/eslint-config-prettier#installation